static_report/src/pages/CRTablesListPage.tsx

```diff
@@ -32,6 +26,8 @@ export function CRTablesListPage({ data }: Props) {
       page: 'table-list-page',
     },
   });
+  const modal = useDisclosure();
```

Comment:
Hmm, I think the common modal should by default spread the disclosure's props inside the component itself, unless there's a case where that isn't desired?

Reply:
Do you mean to create a disclourse inside the common modal?

Reply:
passing `modal` as a prop purpose is we can control the modal from the outside.

Reply:
Ah true. 
And that's necessary for 
```
 <CommonModal {...modal} >
 ```
to work, i presume. 

If that's the case, I don't think `common-modal` is really needed as it's just an extra ui, as it still requires consumer dependencies e.g. `useDisclosure` to work.

Reply:
True. I forgot how disclosure worked again lol